Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -379,13 +377,6 @@
     canCreateVacuum: false
     deleteAfterExplosion: false
     repeatable: true
-  - type: BatterySelfRecharger
-    autoRecharge: true
-    autoRechargeRate: 25
-    autoRechargePause: true
-    autoRechargePauseTime: 50
```

hmmm boooooooooooooooooooooooring, balance the energy shot costs or recharge speed instead :)
```suggestion
  - type: BatterySelfRecharger
    autoRecharge: true
    autoRechargeRate: 25
    autoRechargePause: true
    autoRechargePauseTime: 50